hera_sim/interpolators.py

```diff
@@ -25,6 +25,13 @@ def _check_path(datafile):
 def _read_npy(npy):
     return np.load(_check_path(npy))
 
+# TODO: Update module to have a base `interpolator` class, from which `Tsky`
```

Comment:
I think so. I'd have to relearn good inheritance practices in Python, so it might delay this release a bit, but it's certainly possible (and I should do the thing anyway).